parm/atm/berror/staticb_gsibec.yaml

```diff
@@ -0,0 +1,29 @@
+covariance model: SABER
+full inverse: true
+saber blocks:
+- saber block name: gsi covariance
+  saber central block: true
+  input variables: &control_vars [eastward_wind,northward_wind,air_temperature,surface_pressure,
+                           specific_humidity,cloud_liquid_ice,cloud_liquid_water,
+                           mole_fraction_of_ozone_in_air]
+  output variables: *control_vars
+  gsi akbk: &akbk $(fv3jedi_fix_dir)/akbk.nc4
+  gsi error covariance file: &gsiberr $(fv3jedi_staticb_dir)/gsi-coeffs-gfs-global.nc4
+# gsi error covariance file: &gsiberr $(fv3jedi_staticb_dir)/global_berror.f77
```

Comment:
Are you keeping the comment intentionally?

Reply:
Yes, I kept the `#` line as a reminder that `fv3jedi_var.x` can directly read the binary GSI global_berror file.   The line 

`# gsi error covariance file: &gsiberr $(fv3jedi_staticb_dir)/global_berror.f77`

can easily be removed.   I don't mind do so if you prefer a cleaner look.

Reply:
@RussTreadon-NOAA , no I don't really care, keep it if you need it. I was just wondering if it was an unintended left over.

Reply:
@guillaumevernieres .  It's intended.  I'm old.  I need reminders.  I agree with you, though, that we should drop the binary or netcdf format at some point.  There's no reason to keep two files with the same information.   Since the binary file can be used by `fv3jedi_var.x` and `gsi.x`, I'll keep it around for the time being.

Reply:
Thanks, @guillaumevernieres !